lib/src/test/java/io/strmprivacy/driver/client/StrmPrivacyClientTest.java

```diff
@@ -61,6 +59,7 @@ private Config getTestConfig(WireMockRuntimeInfo wireMockRuntimeInfo) {
                 .gatewayHost("localhost")
                 .gatewayPort(wireMockRuntimeInfo.getHttpPort())
                 .gatewayScheme("http")
+                .keycloakUrl("https://accounts.dev.strmprivacy.io/auth/realms/streams/protocol/openid-connect/token")
```

Comment:
Did you actually update the test itself? You should create the specific mocks with Wiremock in order to properly test it.